### ISSUE TITLE
permissions with creating public ip for ingress

### ISSUE THREAD

**Issue by masyanru** [microsoft/MCW-Containers-and-DevOps#110]
In the task 5 "Configure Kubernetes Ingress" I can deploy ingress controller and I see in kube-system namespace the external ip.
<img width="1137" alt="Screenshot 2019-10-06 at 18 35 58" src="https://user-images.githubusercontent.com/19715692/66271564-3a11d200-e868-11e9-861d-dd9c93b130df.png">

but "az network public-ip list" shows nothing. It shows IPs for aks hosts.

<img width="1222" alt="Screenshot 2019-10-06 at 18 38 32" src="https://user-images.githubusercontent.com/19715692/66271591-8f4de380-e868-11e9-9373-628030fa0037.png">

and nothing new was added in resource group.

<img width="497" alt="Screenshot 2019-10-06 at 18 41 33" src="https://user-images.githubusercontent.com/19715692/66271623-f10e4d80-e868-11e9-8f9b-d36158ae9ce0.png">

I've tried to deploy ingress controller in the default namespace, but it doesn't help.

I think the problem is with permissions for the ingress public IP.

**Comment by masyanru** [comment]
and nothing in default namespace

<img width="1083" alt="Screenshot 2019-10-06 at 18 52 06" src="https://user-images.githubusercontent.com/19715692/66271749-97a71e00-e86a-11e9-95b2-74a4b849a3e0.png">

**Comment by masyanru** [comment]
So you can't run bash script, 'cause if you put public ip for ingress controller in the script, the result of query return nothing and you get this "az network public-ip update: error: argument --ids: expected at least one argument". And it makes sense since you have access only for two public IPs.

**Comment by jamesrcounts** [comment]
The IP address is allocated in the resource group managed by AKS, which by default follows the pattern MC_{aks_resource_group}_{aks_cluster_name}_{region} (if I recall correctly). 

The query you show, should still find the IP regardless of the resource group, but I'm wondering if your account has sufficient rights to see all the resource groups in the subscription.  Are you a subscription scoped Owner or Contributor?

We did a recent end-to-end test for the September update of this lab and have not observed this issue.  We did update the next version of the lab to better explain the expected permissions needed.
